src/main.rs

```diff
@@ -282,6 +294,16 @@ fn get_matches<'a>() -> ArgMatches<'a> {
                          .short("n")
                          .long("name")
                          .help("use name instead of location for find")))
+        .subcommand(SubCommand::with_name("clip")
```

Comment:
maybe it is also worth adding a `-c|--clip` flag to the`show` subcommand, so it can be shown and copied at the same time